src/main/java/util/ReadWriteRequests.java

```diff
@@ -0,0 +1,58 @@
+package util;
+
+import constants.UrlConstants;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.entity.ContentType;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.apache.http.entity.StringEntity;
+
+public class ReadWriteRequests {
+
+    public void makeReadRequest(String token) {
+
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+            HttpGet httpGet = new HttpGet(UrlConstants.URL);
+            httpGet.setHeader("Authorization", "Bearer " + token);
+
+            try (CloseableHttpResponse response = httpClient.execute(httpGet)) {
+                int statusCode = response.getStatusLine().getStatusCode();
+                if (statusCode == 200) {
+                    String responseContent = EntityUtils.toString(response.getEntity());
+                    System.out.println("GET Response: " + responseContent);
```

Comment:
forget about writing anything via print stream. forever. use logger for that goal